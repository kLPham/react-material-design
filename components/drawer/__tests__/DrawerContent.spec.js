import React from 'react';
import { shallow } from 'enzyme';
import DrawerContent from '../drawerContent';

describe('<DrawerContent />', () => {
    it('should render', () => {
        const drawerContent = shallow(
            <DrawerContent>Text</DrawerContent>,
    );
        expect(drawerContent).toMatchSnapshot();
    });
});
