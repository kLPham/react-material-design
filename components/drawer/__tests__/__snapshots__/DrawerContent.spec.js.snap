// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DrawerContent /> should render 1`] = `
<nav
  className="mdc-temporary-drawer__content"
  id="icon-with-text-demo"
>
  Text
</nav>
`;
